### ISSUE TITLE
How is workload_sqls obtained in main.py

### ISSUE THREAD

**Issue by andyyoshikawa0121** [TsinghuaDatabaseGroup/DB-GPT#71]
Hello,

I am trying to understand about the `workload_sqls` that is being fed into main.py. Could you explain to me how these SQLs are being derived?

Is the information being extracted from the pg_stat_statements view?

**Comment by zhouxh19** [comment]
Yes, you can obtain workload_sqls from pg_stat_statements by removing queries over the 'postgres' database (https://github.com/TsinghuaDatabaseGroup/DB-GPT/blob/main/utils/database.py#L330). 

However, as mentioned in README, pg_stat_statements does not support time-period-based filtering and the obtained queries may not be within the anomaly period. A better choice is to (1) directly extract queries from the anomaly trigging scripts (for test only) or (2) utilize advanced tools in commercial databases (e.g., AWR report in Oracle).

Queries can be most critical for finding useful root causes. Thus, if you have any better strategies to extract representative historical queries during a time period (as we know, logging all the queries can be unaffordable). Please let us know -:)

**Comment by andyyosshi** [comment]
Thank you for your detailed response, it was very informative.

Firstly, I appreciate your clear explanation on how to obtain workload_sqls from pg_stat_statements. Your advice on filtering queries on the 'postgres' database was particularly useful.

I also noted your suggestion regarding the issue with time-based filtering. I do not fully understand yet the strategy for extracting representative historical queries over a specific period.

Therefore, I would like to further explore this matter.
Thank you very much.
